### DISCUSSION TITLE
`termux-boot` can be a built-in option

### DISCUSSION THREAD

**Discussion by twaik** [termux/termux-app#2606] [Ideas]
I think `termux-boot` can be built-in into `termux-app`.
`BOOT_COMPLETED` receiver can be optional and disabled in app's manifest by default and can be enabled using pm or `termux-api` command.

What do you think guys?

**Comment by agnostic-apollo** [comment]
That is actually already planned for when something like "systemd" is implemented in termux with events like boot, termux start, stop, shutdown... I guess will just move `termux-boot` code into termux app now. Gonna do that for `termux-styling` too.

Do disable the receiver, a toggle can be added in settings.

**Reply by 010010101** [reply]
Naérm

**Comment by rfiyoy** [comment]
> I think `termux-boot` can be built-in into `termux-app`. `BOOT_COMPLETED` receiver can be optional and disabled in app's manifest by default and can be enabled using pm or `termux-api` command.
> 
> What do you think guys?
